tests/dummy/app/templates/demo/accordion.hbs

```diff
@@ -1,61 +1,58 @@
 <div class="bs-example">
   <!-- BEGIN-SNIPPET accordion-demo -->
-  {{#bs-accordion as |acc|}}
-    {{#acc.item value="1" title="First item"}}
-      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur ea eos hic neque quia sequi sunt. Amet at atque corporis earum et fugiat illum magnam nisi sapiente voluptatem. Laudantium, quam?</p>
-    {{/acc.item}}
-    {{#acc.item value="2" as |aitem|}}
-      {{#aitem.title}}
-        <strong>Second</strong> <i>item</i>
-      {{/aitem.title}}
-      {{#aitem.body}}
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur ea eos hic neque quia sequi sunt. Amet at atque corporis earum et fugiat illum magnam nisi sapiente voluptatem. Laudantium, quam?</p>
-      {{/aitem.body}}
-    {{/acc.item}}
-    {{#acc.item value="3" title="Third item"}}
-      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur ea eos hic neque quia sequi sunt. Amet at atque corporis earum et fugiat illum magnam nisi sapiente voluptatem. Laudantium, quam?</p>
-    {{/acc.item}}
-  {{/bs-accordion}}
+  <BsAccordion as |acc|>
+    <acc.item value="1" @title="First item">
```

Comment:
`value` must be passed as property. It's not a HTML attribute in this case. There are several similar issues in this PR. It's related to this open issue in codemod: https://github.com/rajasegar/ember-angle-brackets-codemod/issues/3

I'm not sure how to address this issue in codemod as I'm not that used to ast. A general rule would be something like: If the property passed in curly invocation syntax is used in template or is used in JavaScript file of the component but not only in `attributeBindings` it must be passed as a property. But not sure if required information is available.